src/internet_identity/src/main.rs

```diff
@@ -597,6 +601,41 @@ fn http_request(req: HttpRequest) -> HttpResponse {
     }
 }
 
+fn security_headers() -> Vec<HeaderField> {
```

Comment:
Can you clarify where you got this list from and what it achieves?

Reply:
Done